webapp/Dockerfile

```diff
@@ -16,22 +16,22 @@ RUN yarn install --production=false
 COPY ./ $WORK_DIR
 
 # setup build environment
-ARG REACT_APP_EOS_API_HOST
-ENV REACT_APP_EOS_API_HOST $REACT_APP_EOS_API_HOST
-ARG REACT_APP_EOS_API_PORT
-ENV REACT_APP_EOS_API_PORT $REACT_APP_EOS_API_PORT
-ARG REACT_APP_EOS_API_PROTOCOL
-ENV REACT_APP_EOS_API_PROTOCOL $REACT_APP_EOS_API_PROTOCOL
-ARG REACT_APP_EOS_CHAIN_ID
-ENV REACT_APP_EOS_CHAIN_ID $REACT_APP_EOS_CHAIN_ID
-ARG REACT_APP_HASURA_URL
-ENV REACT_APP_HASURA_URL $REACT_APP_HASURA_URL
-ARG REACT_APP_MAPBOX_ACCESS_TOKEN
-ENV REACT_APP_MAPBOX_ACCESS_TOKEN $REACT_APP_MAPBOX_ACCESS_TOKEN
-ARG REACT_CAPTCHA_KEY
-ENV REACT_CAPTCHA_KEY $REACT_CAPTCHA_KEY
-ARG REACT_APP_BLOCK_EXPLORER_URL
-ENV REACT_APP_BLOCK_EXPLORER_URL $REACT_APP_BLOCK_EXPLORER_URL
+ARG react_app_eos_api_host
+ENV react_app_eos_api_host $REACT_APP_EOS_API_HOST
```

Comment:
It should be like this `ENV REACT_APP_HASURA_URL $react_app_hasura_url` i.e. during the container compilation, environment variables (ENV in upper case), will be assigned the arguments that are passed by the docker compiler command (args in lower case).